cypress_test/Makefile.am

```diff
@@ -171,6 +207,8 @@ define start_coolwsd_instance
 			--o:user_interface.mode=$(USER_INTERFACE) \
 			--o:accessibility.enable=$(A11Y_ENABLE) \
 			--o:security.enable_macros_execution=true \
+			$(if $(SHORT_IDLE),--o:per_view.out_of_focus_timeout_secs=1) \
+			$(if $(SHORT_IDLE),--o:per_view.idle_timeout_secs=7) \
```

Comment:
We want both, because in IdleHandler I increase frequency of checking if we are inactive to catch that small 1s timeout. I want to make it short as possible in CI.